v0/leanplum/transform.js

```diff
@@ -96,42 +92,7 @@ function responseBuilderSimple(message, category, destination) {
 }
 
 async function startSession(message, destination) {
```

Comment:
This not an async function anymore. We can remove async and the corresponding await.